src/librustdoc/core.rs

```diff
@@ -351,9 +351,9 @@ pub fn run_core(search_paths: SearchPaths,
             };
 
             let send_trait = if crate_name == Some("core".to_string()) {
-                clean::get_trait_def_id(&tcx, &["marker", "Send"], true)
+                clean::path_to_def_local(&tcx, &["marker", "Send"])
             } else {
-                clean::get_trait_def_id(&tcx, &["core", "marker", "Send"], false)
+                clean::path_to_def_local(&tcx, &["core", "marker", "Send"])
```

Comment:
Do `path_to_def` and `path_to_def_local` act the same in this situation? I'm wondering what the motivation here is.

Reply:
Oops should be `path_to_def` here